### PR TITLE
Simplify export csv file

### PR DESCRIPTION
**Pull request by kun2985**
# Description

This PR fusion channels to simplify the export csv file

## Issue (optional)

Issue link: BOT-1306

### CODE REVIEW

File: chaos/models.py [hunk 1]
```diff
@@ -2246,6 +2238,45 @@ def get_client_impacts_between_application_dates(cls, client_id, app_start_date,
...
+        stmt = text(query)
+        stmt = stmt.bindparams(bindparam('client_id', type_=db.String))
+        vars = {}
+        vars['client_id'] = client_id
+        vars['is_visible'] = True
+
+        return db.engine.execute(stmt, vars)
+
+    @classmethod
+    def get_channel_message(cls, channel_id, impact_id):
```
Comment (pthegner) [inline chaos/models.py]: not sure if it's really good to get the text for each channel of one impact (time consuming).

File: chaos/models.py [hunk 2]
```diff
@@ -2246,6 +2238,45 @@ def get_client_impacts_between_application_dates(cls, client_id, app_start_date,
 
         return db.engine.execute(stmt, vars)
 
+    @classmethod
+    def get_client_channels(cls, client_id):
+        query = 'SELECT ch.id,' \
+                '       ch.client_id,' \
```
[the remark this answers is not in the document]
Comment (Fahmus) [reply chaos/models.py]: I have "global name 'self' is not defined" when I use "self.client_id"

[the remark this answers is not in the document]
Comment (pthegner) [reply chaos/models.py]: :thinking:

[the remark this answers is not in the document]
Comment (pthegner) [reply chaos/models.py]: What i mean is : do you really use ch.cliend_id ?

File: chaos/models.py [hunk 3]
```diff
@@ -2246,6 +2252,24 @@ def get_client_impacts_between_application_dates(cls, client_id, app_start_date,
...
         return db.engine.execute(stmt, vars)
 
+    @classmethod
+    def get_client_channels(cls, client_id):
+        query = 'SELECT ch.id,' \
+                '       CONCAT (ch.name,\' (\', string_agg(cht.name::text, \',\'),\')\') AS channel_name ' \
+                'FROM channel AS ch ' \
+                'LEFT JOIN channel_type AS cht ON (cht.channel_id = ch.id) ' \
+                'WHERE ch.client_id = :client_id' \
+                '  AND ch.is_visible = :is_visible ' \
```
Comment (pthegner) [inline chaos/models.py]:
```suggestion
                ' AND ch.is_visible = :is_visible ' \
```

File: chaos/models.py [hunk 4]
```diff
@@ -2246,6 +2252,24 @@ def get_client_impacts_between_application_dates(cls, client_id, app_start_date,
...
+                'GROUP BY ch.id'
+
+        stmt = text(query)
+        stmt = stmt.bindparams(bindparam('client_id', type_=db.String))
```
Comment (pthegner) [inline chaos/models.py]: stmt = stmt.bindparams(bindparam('is_visible', type_=db.Boolean)) ?

**Pull request merged**